tests/async_tests/test_interfaces.py

```diff
@@ -331,3 +331,31 @@ async def test_http_request_unix_domain_socket(uds_server: Server) -> None:
         assert reason == b"OK"
         body = await read_body(stream)
         assert body == b"Hello, world!"
+
+
+@pytest.mark.usefixtures("async_environment")
+@pytest.mark.parametrize("max_keepalive", [1, 3, 5])
+@pytest.mark.parametrize("connections_number", [4])
+async def test_max_keepalive_connections_handled_correctly(
+    max_keepalive, connections_number
```

Comment:
Some missing annotations:
```suggestion
    max_keepalive: int, connections_number: int
```

Reply:
I'll fix it as soon as I reach my laptop

Reply:
If not done already I'd recommend we allow untyped defs in `tests/` (I _think_ we have this tweak in HTTPX, for reference?), so that we're not forced to make add these annotations that often only feel like busy work. :-)

Reply:
> Some missing annotations:
> ```suggestion
>     max_keepalive: int, connections_number: int
> ```

I fixed the typehints (I'm sorry for splitting that into 2 commits, I'm working from the mobile)

Reply:
> If not done already I'd recommend we allow untyped defs in `tests/` (I _think_ we have this tweak in HTTPX, for reference?), so that we're not forced to make add these annotations that often only feel like busy work. :-)

I see that in httpx:
https://github.com/encode/httpx/blob/78cf16ace92b42de5972a490787002631d558b32/setup.cfg#L9-L11

I'll add it to httpcore later.